cypress/support/constants.js

```diff
@@ -20,51 +20,63 @@ export const AuthorityPaths = {
 };
 
 export const USER_NO_ARP = 'test-user-no-arp@test.no';
-
 export const USER_NAME_IN_ARP = 'test-user-name-in-arp@test.no';
-
 export const USER_NO_NAME_IN_ARP = 'test-user-no-name-in-arp@test.no';
-
 export const USER_CONNECT_ARP = 'test-user-connect-arp@test.no';
-
 export const USER_NO_ORCID = 'test-user-no-orcid@test.no';
-
 export const USER_CONNECT_ORCID = 'test-user-connect-orcid@test.no';
-
 export const USER_WITH_AUTHOR = 'test-user-with-author@test.no';
-
 export const USER_SAVE_REGISTRATION = 'test-user-save-registration@test.no';
-
 export const USER_ADD_INSTITUTION = 'test-user-add-institution@test.no';
-
 export const USER_CHANGE_INSTITUTION = 'test-user-change-institution@test.no';
-
 export const USER_INSTITUTION_SUBUNIT = 'test-user-subunit@test.no';
-
 export const USER_INSTITUTION_SUBSUBUNIT = 'test-user-subsubunit@test.no';
-
 export const USER_INSTITUTION_SUBSUBSUBUNIT = 'test-user-subsubsubunit@test.no';
-
 export const USER_WITH_INSTITUTION_REMOVE_INSTITUTION = 'test-user-with-institution-remove-institution@test.no';
-
 export const USER_CURATOR_WITH_AUTHOR = 'test-user-curator@test.no';
-
 export const USER_INST_ADMIN_WITH_AUTHOR = 'test-user-first-inst-admin@test.no';
-
 export const USER_SECOND_INST_ADMIN_WITH_AUTHOR = 'test-user-second-inst-admin-1@test.no';
-
 export const ADMIN_USER = 'test-user-app-admin@test.no';
-
 export const TEST_USER = 'test-user-login@test.no';
-
 export const USER_NO_ROLE = 'test-user-with-no-role@test.no';
-
 export const USER_REMOVE_ORCID = 'test-user-remove-existing-orcid@test.no';
-
 export const USER_NON_CUSTOMER = 'test-user-not-customer@test.no';
-
 export const USER_DRAFT_DOI = 'test-user-draft-doi@test.no';
+export const USER_RESOURCE_TYPE = 'test-user-resource-type@test.no';
+export const USER_VIEW_REGISTRATION = 'test-user-view-registration@test.no';
+
+const userData = {
+  name: '',
+  feideid: true,
+  orcid: true,
+  inArp: true,
+};
 
-export const USER_RESOURCE_TYPE = 'test-user-resource-type@test.no'
-
-export const USER_VIEW_REGISTRATION = 'test-user-view-registration@test.no'
+export const USER = {};
+USER[USER_NO_ARP] = { ...userData, name: 'TestUser, No ARP', feideid: false, orcid: false };
+USER[USER_NAME_IN_ARP] = { ...userData, name: 'TestUser, Name in ARP', feideid: false, orcid: false };
```

Comment:
virker som at `userData` overskrives nesten like ofte som det brukes, så kanskje like greit å bare være eksplisitt for hver bruker?
```
export const mockUserNoArp = {
      id: 'test-user-no-arp@test.no',
      name: 'TestUser, No ARP',
      feideid: false,
      orcid: false,
      inArp: false
};

export const mockUserInArp = {
      id: 'test-user-name-in-arp@test.no',
      name: 'TestUser, Name in ARP',
      feideid: false,
      orcid: false,
      inArp: true
};
```

Reply:
Syns det er lettere å holde oversikt på den måten jeg har skrevet det nå.

Reply:
kan evt flytte alt til en operasjon også
```
const USER = {
   [USER_NO_ARP]: { ...userData, name: 'TestUser, No ARP', feideid: false, orcid: false },
   [USER_NAME_IN_ARP]: { ...userData, name: 'TestUser, Name in ARP', feideid: false, orcid: false },
   (...)
}
```

Reply:
får Java-assosiasjoner av masse UPPERCASING, så kan gjerne bruke camelCase for min del. I JS er uansett `const` en konstant, og `let` en "vanlig" variabel. Sens det er litt enklere å lese, og føles mindre skrikende, men det er kanskje bare en smakssak 😅

Reply:
Er vant til at konstanter er uppdarcase, men kan bytte til camelCase uten problemer

Reply:
Husk at jeg har kodet java i over 20 år :P

Reply:
Tror jeg må lage en egen PR på å endre all uppercase...